commit 318e8e1db5d33d41916126963f039bc230962122
Author: Mark Robert Miller <markrmiller@apache.org>
Date:   Sun Jul 22 23:45:01 2012 +0000

    boost timeout slightly for connections and add another call to wait for a stable cluster before killing a node
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1364463 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/solr/core/src/test/org/apache/solr/cloud/FullSolrCloudTest.java b/solr/core/src/test/org/apache/solr/cloud/FullSolrCloudTest.java
index 2df533c..11b30ae 100644
--- a/solr/core/src/test/org/apache/solr/cloud/FullSolrCloudTest.java
+++ b/solr/core/src/test/org/apache/solr/cloud/FullSolrCloudTest.java
@@ -544,6 +544,9 @@ public class FullSolrCloudTest extends AbstractDistributedZkTestCase {
       
       indexAbunchOfDocs();
       
+      // check again 
+      waitForRecoveriesToFinish(false);
+      
       commit();
       
       assertDocCounts(VERBOSE);
@@ -1404,7 +1407,7 @@ public class FullSolrCloudTest extends AbstractDistributedZkTestCase {
           + DEFAULT_COLLECTION;
       HttpSolrServer s = new HttpSolrServer(url);
       s.setConnectionTimeout(DEFAULT_CONNECTION_TIMEOUT);
-      s.setSoTimeout(15000);
+      s.setSoTimeout(20000);
       s.setDefaultMaxConnectionsPerHost(100);
       s.setMaxTotalConnections(100);
       return s;
